### PR TITLE
Remove placeholder text from eApp

### PR DESCRIPTION
**Pull request by line47**
This PR removes placeholder text from eApp. so we no longer have `###` or `Enter an email address`

Fixes https://github.com/18F/e-QIP-prototype/issues/467
Fixes https://github.com/18F/e-QIP-prototype/issues/800

## Screenshots
screen shot 2018-11-09 at 10 00 52 am
screen shot 2018-11-09 at 10 01 00 am

### CODE REVIEW

File: src/components/Form/Location/Location.jsx [hunk 1]
```diff
@@ -850,18 +850,18 @@ Location.defaultProps = {
...
-  streetPlaceholder: i18n.t('address.us.street.placeholder'),
+  streetPlaceholder: '',
   street2Label: i18n.t('address.us.street2.label'),
   stateLabel: i18n.t('address.us.state.label'),
-  statePlaceholder: i18n.t('address.us.state.placeholder'),
+  statePlaceholder: '',
   cityLabel: i18n.t('address.us.city.label'),
-  cityPlaceholder: i18n.t('address.us.city.placeholder'),
-  zipcodePlaceholder: i18n.t('address.us.zipcode.placeholder'),
+  cityPlaceholder: '',
```
Comment (el-mapache) [inline src/components/Form/Location/Location.jsx]: I _think_ we can remove all of these default placeholder props entirely, if we don't reference them in the components, we shouldn't need a default.

File: src/components/Form/Currency/Currency.jsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ export default class Currency extends React.Component {
 
 Currency.defaultProps = {
   name: 'currency',
-  placeholder: i18n.t('currency.placeholder'),
+  placeholder: '',
```
Comment (el-mapache) [inline src/components/Form/Currency/Currency.jsx]: Lets remove this one as well!

File: src/components/Form/Location/Address.jsx [hunk 3]
```diff
@@ -551,7 +551,7 @@ Address.defaultProps = {
   },
   showPostOffice: false,
   streetLabel: i18n.t('address.us.street.label'),
-  postOfficeStreetPlaceholder: i18n.t('address.apoFpo.street.placeholder'),
+  postOfficeStreetPlaceholder: '',
```
Comment (el-mapache) [inline src/components/Form/Location/Address.jsx]: we can pull this placeholder default out too

File: src/components/Form/Currency/Currency.jsx (hunk 2 above)
Comment (line47) [reply src/components/Form/Currency/Currency.jsx]: Removed!

File: src/components/Form/Location/Address.jsx (hunk 3 above)
Comment (line47) [reply src/components/Form/Location/Address.jsx]: also removed 👢

**Pull request merged**